Header header
uint16 id_anchor
float32 distance